fabric/fabric19/build.gradle

```
@@ -0,0 +1,45 @@
+plugins {
+    id 'fabric-loom' version '0.12-SNAPSHOT'
+}
+
+sourceSets {
+    main {
+        java.srcDirs = ['src']
+        resources.srcDirs = ['resources']
+    }
+}
+
+java.toolchain.languageVersion = JavaLanguageVersion.of(17)
+
+archivesBaseName = 'NegativityFabric-19'
```

```suggestion
archivesBaseName = 'NegativityFabric-1.19'
```

Ca fait bizarre sinon.

De même pour la 1.19.2.